darwin/cli_functions.py

```diff
@@ -637,27 +645,41 @@ def set_file_status(dataset_slug: str, status: str, files: List[str]):
         _error(f"No dataset with name '{e.name}'")
 
 
-def find_supported_format(query, supported_formats):
+def find_import_supported_format(
+    query: str,
+    supported_formats: List[ImporterFormat],
+) -> ImportParser:
+    for (fmt, fmt_parser) in supported_formats:
+        if fmt == query:
+            return fmt_parser
+    list_of_formats = ", ".join([fmt for fmt, _ in supported_formats])
+    _error(f"Unsupported import format, currently supported: {list_of_formats}")
+
+
+def find_export_supported_format(
+    query: str,
+    supported_formats: List[ExporterFormat],
+) -> ExportParser:
     for (fmt, fmt_parser) in supported_formats:
         if fmt == query:
             return fmt_parser
     list_of_formats = ", ".join([fmt for fmt, _ in supported_formats])
-    _error(f"Unsupported format, currently supported: {list_of_formats}")
+    _error(f"Unsupported export format, currently supported: {list_of_formats}")
 
 
-def dataset_convert(dataset_slug: str, format: str, output_dir: Union[str, Path, None] = None):
-    client = _load_client()
-    parser = find_supported_format(format, darwin.exporter.formats.supported_formats)
+def dataset_convert(dataset_slug: str, format: str, output_dir: Union[str, Path, None] = None) -> None:
```

Comment:
What do you think of making a new type
```python
PathLike = Union[str, Path]
```
so here would be
```suggestion
def dataset_convert(dataset_slug: str, format: str, output_dir: Optional[PathLike] = None) -> None:
```

The reason why I think it's important is that we have `Union[str, Path]` all over the place

Reply:
I like it !

Reply:
I think this is an antipattern, the whole goal of types is to provide a unique way to identify what is what, by doing this you violate it. A better strategy is to always use Path where we need a Path instance and string where we need a string.